### ISSUE TITLE
Org switching - issues/observations

### ISSUE THREAD

**Issue by neha00290** [openobserve/openobserve#5141]
### Which OpenObserve functionalities are the source of the bug?

log search

### Is this a regression?

Yes

### Description

- [x] Selecting org, copy share URL, select another org and stream and then pasting URL in another tab shows error on UI and duplicate stream selection appears 
- [x] [object Object] org is displayed after login with base URL
- [x] Getting "Unauthorized Access: You are not authorized to perform this operation, please contact your administrator." error while switching default Org in Pipelines 

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version: v0.13.2-rc1
Commit Hash: 20c062e45e3415042c508bfc36c7eecc76bb1ce0
Build Date: 2024-11-25T07:11:42Z
```

**Comment by xsj321** [comment]
I'm also having problems selecting an org, and when I switch to a non-default org, there's nothing on the screen.
![image](https://github.com/user-attachments/assets/35fa332d-5ebb-4536-9ceb-939f4a13c90e)
This was not a problem in the old version

**Comment by xsj321** [comment]
> I'm also having problems selecting an org, and when I switch to a non-default org, there's nothing on the screen. ![image](https://private-user-images.githubusercontent.com/35422365/390624452-35fa332d-5ebb-4536-9ceb-939f4a13c90e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3NjMyODcsIm5iZiI6MTczMjc2Mjk4NywicGF0aCI6Ii8zNTQyMjM2NS8zOTA2MjQ0NTItMzVmYTMzMmQtNWViYi00NTM2LTljZWItOTM5ZjRhMTNjOTBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI4VDAzMDMwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YmMzYzg5YzQ2MWEzMGMwNTY2MjI1MGVhYjQ0OWIwMzQyMzY1ZTU5ODU5NmM5MmVjZDVmMmQ1ODhlMjQ1NjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.BB-eaigSfrzeX3d9QKlgaki5A0OqEEk8sRdTNYxgdsI) This was not a problem in the old version

I solved this problem after upgrading v0.13.2-rc2
